conf/locale/pt-br/LC_MESSAGES/wm-django.po

```diff
@@ -873,119 +838,118 @@ msgstr ""
 #: lms/envs/common.py
 #, python-brace-format
 msgid "Follow {platform_name} on Instagram"
-msgstr ""
+msgstr "Seguir o {nome da plataforma} do Instagram\n\n\n{platform_name}"
 
 #. Translators: This is the website name of www.twitter.com.  Please
 #. translate this the way that Twitter advertises in your language.
 #: lms/envs/common.py
+#, optional
 msgid "Twitter"
-msgstr ""
+msgstr "Twitter"
 
 #: lms/envs/common.py
 #, python-brace-format
 msgid "Subscribe to the {platform_name} subreddit"
-msgstr ""
+msgstr "Se inscrever\n  para o subreddit {nome da plataforma{platform_name}}"
 
 #: cms/envs/bok_choy.py cms/envs/test.py
 msgid "Your Platform 𝓢𝓽𝓾𝓭𝓲𝓸"
-msgstr ""
+msgstr "Sua Plataforma 𝓢𝓽𝓾𝓭𝓲𝓸"
 
 #: lms/envs/common.py
 #, python-brace-format
 msgid "Subscribe to the {platform_name} YouTube channel"
-msgstr ""
+msgstr "Se inscrever\n  para o canal do YouTube {nome da plataforma}\n\n{platform_name}"
 
 #: lms/envs/common.py
 #, python-brace-format
 msgid "Follow {platform_name} on Twitter"
-msgstr ""
+msgstr "Seguir\n   {plataforma nome} no Twitter\n\n{platform_name}"
 
 #: cms/envs/bok_choy.py lms/envs/bok_choy.py lms/envs/test.py
 msgid "Open édX Platform"
-msgstr ""
+msgstr "Plataforma aberta édX"
 
 #: lms/envs/common.py
 msgid "Credit"
-msgstr ""
+msgstr "Créditos"
 
 #: lms/envs/common.py
 msgid "Know your user identifier"
-msgstr ""
+msgstr "Saber\n  seu identificador de usuário"
 
 #: lms/envs/common.py
 msgid "No Prerequisites"
-msgstr ""
+msgstr "Sem pré-requisitos"
 
 #. Translators: This is the website name of www.meetup.com.  Please
 #. translate this the way that MeetUp advertises in your language.
 #: lms/envs/common.py
 msgid "Meetup"
-msgstr ""
+msgstr "Meetup"
 
 #: cms/envs/common.py lms/envs/common.py
 msgid "Your Platform Description Here"
-msgstr ""
+msgstr "Seu\n  Descrição da plataforma aqui"
 
 #: lms/templates/logout.html
 #, python-brace-format
-msgid ""
-"If you are not redirected within 5 seconds, {start_anchor}click here to go "
-"to the home page{end_anchor}."
-msgstr ""
+msgid "If you are not redirected within 5 seconds, {start_anchor}click here to go to the home page{end_anchor}."
+msgstr "Se\n  você não for redirecionado em 5 segundos, {start_anchor}clique aqui para ir para a página inicial{end_anchor}."
 
 #: cms/envs/common.py lms/envs/common.py
 msgid "Your Platform Name Here"
-msgstr ""
+msgstr "Seu\n  Nome da plataforma aqui"
 
 #: cms/envs/common.py
 msgid "Open after 3 months"
-msgstr ""
+msgstr "Abrir\n  após 3 meses"
 
 #: lms/envs/common.py
 msgid "Know your email address"
-msgstr ""
+msgstr "Saber\n  Seu endereço de email"
 
 #: lms/envs/common.py
 msgid "Course Type"
-msgstr ""
+msgstr "Curso\n  Tipo"
 
 #: lms/envs/common.py
 msgid "Kosovo"
-msgstr ""
+msgstr "Kosovo"
 
 #: cms/envs/bok_choy.py cms/envs/test.py
 msgid "𝓢𝓽𝓾𝓭𝓲𝓸"
-msgstr ""
+msgstr "𝓢𝓽𝓾𝓭𝓲𝓸"
 
```

Comment:
`𝓢𝓽𝓾𝓭𝓲𝓸` is causing problems as well. Maybe there is some character in it with different encoding. If the translation is the same as the id it be left empty.